app/templates/components/public/call-for-speakers.hbs

```diff
@@ -38,7 +38,11 @@
   {{/unless}}
 <div class="ui hidden divider"></div>
 {{#if this.isNewSession}}
-  <p>{{t 'No session have been submitted'}}</p>
+  {{#if this.authManager.currentUser.isVerified}}
```

Comment:
If user is verified, and they see the message `Login to show your submitted sessions`, it'd be confusing

Reply:
if user is verified, they would see **No sessions have been submitted**

Reply:
Sorry, I meant if user is unverified